xcode/Subconscious/Shared/Components/Common/EntryRow.swift

```diff
@@ -14,10 +14,14 @@ struct EntryRow: View, Equatable {
     var emptyExcerpt = "Empty"
 
     var body: some View {
-        VStack(alignment: .leading, spacing: AppTheme.unitHalf) {
-            ExcerptView(excerpt: entry.excerpt, spacing: AppTheme.unitHalf)
-                .font(.callout)
-                .multilineTextAlignment(.leading)
+        VStack(alignment: .leading, spacing: AppTheme.unit) {
+            ExcerptView(
+                excerpt: entry.excerpt,
+                // TODO: Should we be able to tap links from previews in the notebook?
+                onViewSlashlink: { _ in }
```

Comment:
As the comment says, unsure about this.

Reply:
For context:

- Notebook view should be an iOS list view. The entire row is a tappable button that will take you to the full view when tapped.
- Feed view and profile view display stories. These are like social stories. Tapping on the story will take you to the detail view. Tapping on a link in the story will follow the link.

Reply:
We will probably need different views for the notebook list view and the feed/profile list views.